### PR TITLE
Improve contributing.md

### LINKED ISSUE #132: Contribution information's
There is no `CONTRIBUTE.md` file in the repo or any other information about how to contribute.

My question goes in the direction of your workflow? When creating a PR to which branch should I create it? `devel` or should I create my own branch in your repo e.g. `fix_xyz`? Do you have any naming conventions for branches?

Which branch should I "fork" (I hate that word)?

### PR DESCRIPTION
**Pull request by Codeberg-AsGithubAlternative-buhtz**
Fix #132 and #134.

Added some hints, infos and links
 - No quality restrictions. All PR's welcome. ;)
 - Coding guideline incl. linters
 - Use existing test cases.
 - Create new test cases.
 - "development install" via virtual environment or --editable

What is missing?
The column width. Your code use more then 80 chars per line. This is against PEP8 and should be mentioned explicit.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,2 +1,39 @@
...
+
+Of course there are some steps that could be considered to improve the quality of the pull request and to support the process of review and development.
+
+- Make sure your code follows [PEP 8 — Style Guide for Python Code](https://pep8.org/). You can use some [linting tools](https://en.wikipedia.org/wiki/Lint_(software)) to automate the process (e.g. [`pycodestyle`](https://pycodestyle.pycqa.org) or [`flake8`](https://github.com/pycqa/flake8)).
+- Test your code using the existing test cases. For details read further.
+- It would be great if you could create [`unittest`](https://docs.python.org/3/library/unittest.html)'s for your code.
+
+# Unittesting
+
+This project uses pythons default [`unittest`](https://docs.python.org/3/library/unittest.html) package. You can run all test cases via the *discover* feature when you run this in the projects root folder.
```
Comment (DeepSpace2) [inline .github/CONTRIBUTING.md:17]:
```suggestion
This project uses Python's default [`unittest`](https://docs.python.org/3/library/unittest.html) package. You can run all test cases via the *discover* feature when you run this in the projects root folder.
```

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -1,2 +1,39 @@
...
+
+The output should look like this
+```sh
+----------------------------------------------------------------------
+Ran 74 tests in 0.823s
+
+OK
+```
+
+If you don't want to run all tests at once but a specific test case or method you need to *install* the package first. It is possible via a virtual environment but we recommend to use the `--editable` switch of `pip`. Run this in the projects root folder:
```
Comment (DeepSpace2) [inline .github/CONTRIBUTING.md:31]:
```suggestion
If you want to run a specific test case or method you need to *install* the package first. It is recommended to use a virtual environment and the  `--editable` flag of `pip`. Run this in the projects root folder:
```

**Comment by DeepSpace2** [comment]
Thank you for this PR! It is much appreciated. Please see the small suggestion for changes that I've made.

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (Codeberg-AsGithubAlternative-buhtz) [reply .github/CONTRIBUTING.md:31]: It is the first time I use the review feature of GitHub. Is this the right way doing thins? ;)

You suggest "virtual environment and the".
But I meant "virtual environment or the".

Comment (DeepSpace2) [reply .github/CONTRIBUTING.md:31]: I know, I deliberately used "and the". These are not mutually exclusive, and not using a virtual environment will prevent one from using both a public/released version of the library and the development (local) version.

**Comment by Codeberg-AsGithubAlternative-buhtz** [comment]
OK, I see everything "resolved" but GitHub tells me "2 workflows awaiting approval". Do I have to do something else for this PR?

**Comment by DeepSpace2** [comment]
Merged. Thanks for contributing :)

**Pull request merged**